test/orswot.rs

```diff
@@ -16,7 +17,7 @@ fn build_opvec(op_prims: Vec<(u8, u8, u8, u64)>) -> OpVec {
     for (actor, member, choice, counter) in op_prims {
         let op = match choice % 2 {
             0 => Op::Add {
-                member,
+                members: once(member).collect(),
```

Comment:
These tests as they are, is not testing for any new behavior that might have been introduced in with this change.

It should be fairly straightforward to get quickcheck to give us randomized `member`'s here.

Changing the type signature of the quickcheck tests to ask for a `Vec<u8>` instead of `u8` for member's should give you the data you need to generalize this test for many members.
``` rust
    // then you can change the op_prims type to this:
    fn build_opvec(op_prims: Vec<(u8, Vec<u8>, u8, u64)>) -> OpVec {
    //  instead of
    fn build_opvec(op_prims: Vec<(u8, u8, u8, u64)>) -> OpVec {
```

Let me know if I'm not being clear here, I know this bit of the code is a bit more hairy :|

Reply:
Thanks for the pointers -- I tried `HashSet<u8>` instead of `Vec<u8>` and it seems to work.